data/src/main/java/io/quarkus/qe/data/marshallers/QuarkusExtensionMarshaller.java

```diff
@@ -11,6 +11,7 @@
     public QuarkusExtension fromEntity(QuarkusExtensionEntity entity) {
         QuarkusExtension model = new QuarkusExtension();
         model.setName(entity.name);
+        model.setVersion(entity.version.id);
```

Comment:
Are we 100% sure that the version will be always populated? We are checking whether is null in the repository marshaller.

Reply:
no can be null!, I am going to double check!